pkg/scannerv4/mappers/mappers.go

```diff
@@ -307,10 +308,76 @@ func toProtoV4PackageVulnerabilitiesMap(ccPkgVulnerabilities map[string][]string
 		pkgVulns[id] = &v4.StringList{
 			Values: filterRepeatedVulns(vulnIDs, ccVulnerabilities),
 		}
+		pkgVulns[id].Values = sortBySeverity(pkgVulns[id].GetValues(), vulnerabilities)
 	}
 	return pkgVulns
 }
 
+// getMaxBaseScore returns the highest CVSS base score found in the CVSS metrics for V3 and V2,
+// prioritizing scores from the Red Hat source. If no Red Hat score is found, it returns the highest available score.
+func getMaxBaseScore(cvssMetrics []*v4.VulnerabilityReport_Vulnerability_CVSS) float32 {
```

Comment:
I think we should just use the first one (or the value of `vuln.GetCvss()` even though it's (externally) deprecated), as the first one is defined to be the StackRox "preferred" score. That way there'd be no need to have the Red Hat check, either. It's just a generic max func based on the StackRox "preferred" scores